react/features/base/connection/functions.js

```diff
@@ -54,7 +56,17 @@ export function getInviteURL(stateOrGetState: Function | Object): string {
         throw new Error('Can not get invite URL - the app is not ready');
     }
 
-    return getURLWithoutParams(locationURL).href;
+    const { inviteDomain } = state['features/dynamic-branding'];
+    const urlWithoutParams = getURLWithoutParams(locationURL);
+
+    if (inviteDomain) {
+        const meetingId
+            = config.brandingRoomAlias || urlWithoutParams.pathname;
```

Comment:
Please use `state['features/base/config']` to retrieve the `brandingRoomAlias` value.

Reply:
done